components/wsdl-tools/wsdlvalidator/org.wso2.carbon.tools.wsdlvalidator.ui/src/main/resources/web/wsdl_validator/index.jsp

```diff
@@ -73,7 +73,7 @@
             document.getElementById("fileResult").innerHTML = value;
         }
 
-        if (value.search(/INVALID/g) == -1) {
+        if (value == "WSDL DOCUMENT IS VALID") {
```

Comment:
Why do we use equal operator here? Might need to use value.equals():
http://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java